pkg/testsuite/mock/wallet_transactions.go

```diff
@@ -631,6 +665,46 @@ func (w *Wallet) CreateNFTFromInput(transactionName string, inputName string, op
 	)
 }
 
+//nolint:forcetypeassert
+func (w *Wallet) CreateNativeTokenFromInput(transactionName string, inputName string, accountOutputName string) *iotago.SignedTransaction {
+	input := w.Output(inputName)
+	accountOutput := w.AccountOutput(accountOutputName)
+	mintedAmount := input.BaseTokenAmount()
```

Comment:
Doesn't need to be the same amount, maybe pass that as a parameter to the function?